databricks/koalas/testing/utils.py

```diff
@@ -32,21 +32,19 @@
 
 
 class PySparkTestCase(unittest.TestCase):
-
```

Comment:
I'd prefer to have a blank line between the class declaration and its first member. Is it possible?

Reply:
emm... I am afraid not

Reply:
@ueshin  but there is a walkaround, if we add docstring, could be one sentence only:
```python
class PySparkTestCase(TestCase):
    """pyspark customize test case"""
    def __init__(args):
         pass
```
then, Black will automatically add a blank line in between:
```python
class PySparkTestCase(TestCase):
    """pyspark customize test case"""

    def __init__(args):
         pass
```